### PR TITLE
MJX model for Unitree Go2

### PR DESCRIPTION
**Pull request by lonelyfluency**
Modified Go2 MJCF for mjx training.
go2_n

### CODE REVIEW

**Comment by lonelyfluency** [comment]
CLA signed.

**Comment by kevinzakka** [comment]
Nice @lonelyfluency!

File: unitree_go2/README.md [hunk 1]
```diff
@@ -30,6 +30,18 @@ to OBJ format using [Blender](https://www.blender.org/).
...
+2. frictionloss was removed.
+3. The friction cone was changed from elliptic to pyramidal.
+4. All contacts other than the sphere geoms are changed to sphere geoms and placed to the joints position to simulate collisions at critical locations.
+
 ## License
 
 This model is released under a [BSD-3-Clause License](LICENSE).
+
+## Acknowledgments
+The MJX model of Go2 was contributed by lonelyfluency.
```
Comment (kevinzakka) [inline unitree_go2/README.md:47]: Add a newline in between the title and the sentence.

File: unitree_go2/README.md [hunk 2]
```diff
@@ -30,6 +30,18 @@ to OBJ format using [Blender](https://www.blender.org/).
    increased `impratio` to reduce slippage.
 8. Added `scene.xml` which includes the robot, with a textured groundplane, skybox, and haze.
 
+## MJX
+
+A version of the Unitree Go2 model for use in MJX is available in scene_mjx.xml with the following changes:
```
Comment (kevinzakka) [inline unitree_go2/README.md:35]: Can you enclose all the variables such as `scene_mjx.xml` and `frictionloss` with ``?

File: unitree_go2/README.md [hunk 3]
```diff
@@ -30,6 +30,18 @@ to OBJ format using [Blender](https://www.blender.org/).
...
+
+1. The solver iterations and ls_iterations were modified for performance.
+2. frictionloss was removed.
+3. The friction cone was changed from elliptic to pyramidal.
+4. All contacts other than the sphere geoms are changed to sphere geoms and placed to the joints position to simulate collisions at critical locations.
```
Comment (kevinzakka) [inline unitree_go2/README.md:40]: Can you explain this a bit better @lonelyfluency?

Comment (kevinzakka) [reply unitree_go2/README.md:40]: Maybe having a picture of the collision geoms as a PNG and adding it to this section might be worthwile?

**Review by kevinzakka — CHANGES_REQUESTED** [review]
Thanks @lonelyfluency, left some comments!

**Review by kevinzakka — APPROVED** [review]

**Comment by lonelyfluency** [comment]
Thank you @kevinzakka, very good comments.

**Comment by kevinzakka** [comment]
Merged, thank you @lonelyfluency!

**Pull request closed without merging**